src/app/components/list/toolbar/index.html

```diff
@@ -128,13 +128,44 @@ <h3>
   </sky-demo-page-example>
 	
 	<sky-demo-page-content sectionHeading="UX guidelines">
-		<div class="sky-subsection-heading">Sorting</div>
-		<p>If you use the toolbar with a grid view, you have two options for letting users sort the list:</p>
-		<ul>
-			<li>The sort button in the toolbar</li>
-			<li>Clickable column headers</li>
-		</ul>
-		<p>Don't combine both sorting options in a single list because it confuses users about which option is currently applied.</p>
+		
+		<section>
+			<header><div class="sky-section-heading">Usage</div></header>
+			<stache-row>
+				<stache-column screenSmall="6">
+					<p>
+						The list toolbar appears above a <a routerLink="/components/list">list</a> of records. It contains actions to manipulate the contents and display of the list. Place frequently used actions directly in the toolbar, and place less-frequently used actions in a more actions menu. When multiple views of the list are available, the view switcher is docked on the right side of the toolbar.
```

Comment:
I know this text was already in place on the toolbar component docs, but reading through it again, "It contains actions to manipulate the contents and display of the list" reads awkwardly. At least the "display of the list" part does. I'd change "display" to "appearance" unless we mean something else by "display" here.

Reply:
"Display" is a bit awkward but has a slightly broader interpretation than "appearance" which implies visual aspects, which aren't actually changeable. The intent is to connote things like sorting and view switching - I don't think "appearance" is correct but if you have ideas other than "display" I'm happy to make a switch.

Reply:
Would "organization" work? ... The problem with "display" is that I initially read it as a verb, not a noun, so I have to read the sentence twice to understand it.

Reply:
I'm wondering if we can just get rid of "and display" - referring to "contents" seems broad enough to cover everything.

Reply:
That makes sense to me.